src/arch/arm/ArmTLB.py

```diff
@@ -45,6 +45,26 @@ class ArmLookupLevel(Enum):
     vals = ["L0", "L1", "L2", "L3"]
 
 
+class TLBIndexingPolicy(SimObject):
+    type = "TLBIndexingPolicy"
+    abstract = True
+    cxx_class = "gem5::IndexingPolicyTemplate<gem5::ArmISA::TLBTypes>"
+    cxx_header = "arch/arm/pagetable.hh"
+    cxx_template_params = ["class Types"]
+
+    # Get the size from the parent (cache)
+    size = Param.Int(Parent.size, "number of TLB entries")
```

Comment:
In other contexts of indexing policies, size is defined to be MemoryType("8KB", "16KB" etc.,). Can we rename this to 'numEntries' to avoid confusion?

Reply:
Makes sense. I was using size for consistency with the Tlb field but I guess there's nothing wrong in doing

`num_entries = Param.Int(Parent.size, "number of TLB entries")`